src/shell/pipe_exec/mod.rs

```diff
@@ -495,7 +497,11 @@ impl PipelineExecution for Shell {
                     self.watch_foreground(child.id(), child.id(), move || long, |_| ())
                 }
                 Err(e) => if e.kind() == io::ErrorKind::NotFound {
-                    eprintln!("ion: command not found: {}", short);
+                    if let Some(output) = command_not_found(self, &short) {
```

Comment:
Is `short` the wrong variable to use here? Am I being insecure?

Reply:
Could use `&job.command`

Reply:
Then again, I guess `short` is the only option here.

Reply:
Awesome, thanks!